documentation/user-consent.md

```diff
@@ -0,0 +1,169 @@
+
+>[TO BE EDITED/COMPLETED]
+
+### Purpose relationship with other key concepts
+
+The key concepts defined to handle user personal information are related to purpose concept in the following way:
+
+![Key concepts relationship](./images/key-concepts-relationship.png)
+
+- Scopes tell what resources (e.g. APIs) are accessible under those scopes.
+- A purpose has associated a set of personal information scopes (PI scopes). 
+  - These are scopes that refer to resources providing access to Personal Information and thus requiring special protection as stated by e.g. the GDPR law.
+  - Getting access to PI scopes MUST always be done by explicitly declaring a purpose.
+- Purposes are assigned to an application.
+
+Therefore, an application will have access to:
+  - APIs or resources providing personal information under the assigned purpose (because of the PI scopes included in that purpose)
+    - **Only** if user has granted her consent for that purpose
+    - Which may happen automatically (automatic purposes) or may need an explicit user action to provide consent (depending on the purpose level of the purpose associated to that consent). In the second case, the consent is created and stored and will be validated during authorization.
+
+>[TO BE EDITED/COMPLETED]
+
+### Purpose definition
+
+>[TO BE EDITED/COMPLETED] - purpose definition (naming + description) and format. Data Privacy Vocabulary: https://www.w3.org/community/dpvcg/​ | https://w3c.github.io/dpv/dpv/#vocab-purpose
+
+### Applying purpose concept in the authorization request
+
+>[TO BE EDITED/COMPLETED - [OAuth 2.0 Authorization Framework](https://www.rfc-editor.org/rfc/rfc6749)]
+
+## User Identity
+
+>[TO BE EDITED/COMPLETED] - User Identification: Get an identifier of a user by some piece of information. E.g.: IP, phone number, document number...
+
+## User Authentication/Authorization & Consent Management
+
+>[TO BE EDITED/COMPLETED] - The base standard is OpenID Connect. User authentication: Verify that the user is who they say they are. For example: IP/Network Authentication, SMS OTP, User/Password, Push to Operator App... it depends on the type of authorisation (grant type) chosen according to the application use cases, integration model and requirements (B2B/B2C, via aggregator or not, delegated consent or not...). User authorisation: Verify access to the requested resource (OpenID Connect scope, purpose) through policies and rules... user consent is checked at this point as a pre-requisite before issuing an access token.
+
+### Authentication mechanism/s
+
+>[TO BE EDITED/COMPLETED] - Authentication mechanism/s to be considered... E.g.: IP/Network Authentication, SMS OTP, User/password, Push to Operator App​...
+
+### Consent capture and storage
+
+>[TO BE EDITED/COMPLETED] - Consent capture (by Operator vs by API Client a.k.a delegated) and storage (in Operator vs in a third party).
+
+### Athorization flows / grant types
```

Comment:
```suggestion
### Authorization flows / grant types
```

Reply:
fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/13/commits/5cb8ed958e0cba62bc76a8c610b7679bba3016ff